core/app-framework/app-native-shared/bi-inc/attr_container.h

```diff
@@ -20,13 +20,17 @@ extern "C" {
 
 /* Attribute type */
 enum {
-    ATTR_TYPE_BEGIN = 1,
-    ATTR_TYPE_SHORT = ATTR_TYPE_BEGIN,
+    ATTR_TYPE_BEGIN = 0,
+    ATTR_TYPE_BYTE = ATTR_TYPE_BEGIN,
+    ATTR_TYPE_INT8 = ATTR_TYPE_BYTE,
+    ATTR_TYPE_SHORT,
     ATTR_TYPE_INT,
     ATTR_TYPE_INT64,
-    ATTR_TYPE_BYTE,
+    ATTR_TYPE_UINT8,
     ATTR_TYPE_UINT16,
-    ATTR_TYPE_FLOAT,
+    ATTR_TYPE_UINT32,
+    ATTR_TYPE_UINT64,
+    ATTR_TYPE_FLOAT = 10,
```

Comment:
Why set `ATTR_TYPE_FLOAT` to 10? Seems unnecessary?

Reply:
We determine the number of bytes that should be copied through `1<<(type & 3)` (in `attr_container.c:618`).
`ATTR_TYPE_BYTE = 0`, so the number of bytes is `1 << 0  = 1`.
`ATTR_TYPE_UINT64 = 7`, so the number of bytes is `1 << 3 = 8`
The `ATTR_TYPE_FLOAT` need to be `10`, because of `1 << 10&3 = 1 << 2 = 4`. The number of bytes of float is `4`.
If we don't use this `&3` mechanism, we need to change the code that handles the number of bytes.

Reply:
OK, got it. So defining it as 10 is OK, could you add comments for it?

Reply:
I got it. Thank you. I will add comments for it.